common/transport/mqtt/test/_mqtt_base_test.js

```diff
@@ -14,9 +14,24 @@ var Message = require('azure-iot-common').Message;
 var endpoint = require('azure-iot-common').endpoint;
 var errors = require('azure-iot-common').errors;
 
+
+
+
+
```

Comment:
ok super nitpicky but watch out for unnecessary whitespace? I know, i'm nitpicky. sorry.